core/src/main/java/org/lint/azzert/context/Context.java

```diff
@@ -16,6 +16,7 @@
     private final Set<String> assertApis;
     private final Set<String> testFrameworks;
     private final Set<String> exemptApis;
+    private final Set<String> supportedParameters;
```

Comment:
How/Where are the supported params being used?

Reply:
Supported parameters currently holds only "enabled" for testNG, I didnt want to use test ng specific code which is why added a supported parameters, currently supported parameter can only be true or false.